package.json

```diff
@@ -5,15 +5,11 @@
     "@babel/cli": "^7.1.5",
     "@babel/core": "^7.2.0",
     "@babel/plugin-proposal-class-properties": "^7.2.1",
-    "@babel/plugin-proposal-export-default-from": "^7.0.0",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.2.0",
```

Comment:
we use `transform-runtime` in rollup.config.js

Reply:
Ah shoot I totally missed that. For some reason the build still works without failing. 🤔